linode/manager
# feat: [M3-7253] - Add AGLB details - Configurations - Routes Table

## Description 📝
Add AGLB details - Configurations - Routes Table


| Before  | After   |
| ------- | ------- |
|![image](https://github.com/linode/manager/assets/119517080/6a7b48a2-3d01-45e9-9d8a-a9911ee84df0)|![image](https://github.com/linode/manager/assets/119517080/b905876f-6e5a-49bc-b634-9333874ddc76)|

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Checkout the branch and run the app locally 
- Enable MSW 

### Verification steps 
(How to verify changes)
- Navigate to http://localhost:3000/loadbalancers/0/configurations
- Click on any configuration accordion.
- Verify expandable Routes Table in Configurations Tab. 
- Verify expandable Routes Table in Routes Tab

## As an Author I have considered 🤔

*Check all that apply*
